server/journeys/referral/check-answers/controller.ts

```diff
@@ -20,6 +22,52 @@ export class ReferralCheckAnswersController {
     )
   }
 
+  checkSubmitToAPI: RequestHandler = async (req, res, next) => {
+    const prisoner = req.journeyData.prisoner!
+    const referral = req.journeyData.referral!
+    try {
+      await this.createReferral(req, {
+        logNumber: prisoner.prisonId,
+        referral: {
+          contributoryFactors: referral.contributoryFactors!.map(factor => ({
+            factorTypeCode: factor.factorType.code,
+            ...(factor.comment !== undefined && { comment: factor.comment }),
+          })),
+          incidentDate: referral.incidentDate!,
+          incidentLocationCode: referral.incidentLocation!.code,
+          incidentTypeCode: referral.incidentType!.code,
+          refererAreaCode: referral.refererArea!.code,
+          referredBy: referral.referredBy!,
+          ...(typeof referral.assaultedStaffName === 'string' && { assaultedStaffName: referral.assaultedStaffName }),
+          descriptionOfConcern: referral.descriptionOfConcern!,
+          incidentInvolvementCode: referral.incidentInvolvement!.code,
+          incidentTime: `${referral.incidentTime!}:00`,
+          isProactiveReferral: referral.isProactiveReferral!,
+          // @ts-expect-error asd
+          isSaferCustodyTeamInformed: 'YES',
```

Comment:
It's not passing through the right enum value here. I noticed a [TODO about replacing the enum](https://github.com/ministryofjustice/hmpps-challenge-support-intervention-plan-ui/blob/main/server/%40types/csip/csipApiTypes.ts#L7) with the new `OptionalYesNoAnswer`.

In Appointments, we [exposed the enum as a global in nunjucks](https://github.com/ministryofjustice/hmpps-activities-management/blob/main/server/nunjucks/nunjucksSetup.ts#L192) which [allowed its use in the view rather than strings](https://github.com/ministryofjustice/hmpps-activities-management/blob/main/server/views/pages/appointments/create-and-edit/repeat.njk#L33)

Reply:
Ah crap sorry - I meant to do this as a last thing but completely forgot - thank you.